torchbenchmark/models/pytorch_unet/pytorch_unet/.gitignore

```diff
@@ -0,0 +1,8 @@
+*.pyc
```

Comment:
Please manually remove this file and the `.github` directory as we are managing gitignores globally.

Reply:
Will do.